narwhals/series.py

```diff
@@ -440,6 +440,42 @@ def std(self, *, ddof: int = 1) -> Any:
         return self._series.std(ddof=ddof)
 
     def is_in(self, other: Any) -> Self:
+        """
+        Check if elements of this Series are in the other Series.
+
+        Arguments:
+            other: Series or sequence of primitive type.
```

Comment:
this should just be "sequence of primitive type"

Reply:
thanks, I corrected the description of `other`